src/main/java/kitchenpos/application/OrderService.java

```diff
@@ -20,91 +29,89 @@
+                orderTableRepository.findById(orderRequest.getOrderTableId())
+                                    .orElseThrow(() -> new NotFoundOrderTableException("해당 주문 테이블이 존재하지 않습니다."));
+        final List<OrderLineItem> orderLineItems = convertToOrderLineItem(orderRequest);
+        final Order order = orderRequest.toEntity(orderTable, OrderStatus.COOKING, LocalDateTime.now(), orderLineItems);
+        final Order savedOrder = orderRepository.save(order);
 
-        final List<Long> menuIds = orderLineItems.stream()
-                .map(OrderLineItem::getMenuId)
-                .collect(Collectors.toList());
+        return OrderResponse.from(savedOrder);
+    }
 
-        if (orderLineItems.size() != menuDao.countByIdIn(menuIds)) {
-            throw new IllegalArgumentException();
+    private void validateOrderLineItemsIsExists(final List<OrderLineItemDto> orderLineItemDtos) {
+        if (orderLineItemDtos.size() != menuRepository.countByIdIn(convertToIds(orderLineItemDtos))) {
+            throw new NotFoundOrDuplicateMenuToOrderExcpetion("존재하지 않는 혹은 중복된 메뉴를 주문 항목으로 설정했습니다.");
         }
+    }
 
-        order.setId(null);
-
-        final OrderTable orderTable = orderTableDao.findById(order.getOrderTableId())
-                .orElseThrow(IllegalArgumentException::new);
+    private List<OrderLineItem> convertToOrderLineItem(final OrderRequest orderRequest) {
+        final List<OrderLineItem> savedOrderLineItems = new ArrayList<>();
 
-        if (orderTable.isEmpty()) {
-            throw new IllegalArgumentException();
+        for (final OrderLineItemDto orderLineItemDto : orderRequest.getOrderLineItems()) {
+            final Menu menu = menuRepository.findById(orderLineItemDto.getMenuId())
+                                            .orElseThrow(() -> new NotFoundMenuException("해당 메뉴가 존재하지 않습니다."));
+            final OrderLineItem orderLineItem = orderLineItemDto.toEntity(null, menu);
+            savedOrderLineItems.add(orderLineItem);
         }
 
-        order.setOrderTableId(orderTable.getId());
-        order.setOrderStatus(OrderStatus.COOKING.name());
-        order.setOrderedTime(LocalDateTime.now());
-
-        final Order savedOrder = orderDao.save(order);
-
-        final Long orderId = savedOrder.getId();
-        final List<OrderLineItem> savedOrderLineItems = new ArrayList<>();
-        for (final OrderLineItem orderLineItem : orderLineItems) {
-            orderLineItem.setOrderId(orderId);
-            savedOrderLineItems.add(orderLineItemDao.save(orderLineItem));
-        }
-        savedOrder.setOrderLineItems(savedOrderLineItems);
+        return savedOrderLineItems;
+    }
 
-        return savedOrder;
+    private static List<Long> convertToIds(final List<OrderLineItemDto> orderLineItemDtos) {
+        return orderLineItemDtos.stream()
+                                .map(OrderLineItemDto::getMenuId)
+                                .collect(Collectors.toList());
     }
```

Comment:
```suggestion
    private List<Long> convertToIds(final List<OrderLineItemDto> orderLineItemDtos) {
        return orderLineItemDtos.stream()
                                .map(OrderLineItemDto::getMenuId)
                                .collect(Collectors.toList());
    }
```

이런 부분은 단순 실수니까 (제이미가 잘해놓으셔서 더 없을 수도 있지만 ㅋㅋ) 개행과 같은 실수 별도 코멘트를 달진 않을게요!!

Reply:
헉..!
지금 보니 의미 없이 `static`이 몇 군데 붙어 있었군요...!
감사합니다.